bin-src/main.js

```diff
@@ -70,7 +70,7 @@ export async function runBuild(ctx) {
     ctx.options = await getOptions(ctx);
   } catch (e) {
     ctx.log.info('');
-    ctx.log.error(e.message);
+    ctx.log.error(e.stack || e.message);
```

Comment:
This should be wrapped in `fatalError` like below. Just throwing a stack trace at the user isn't good UX.